tools/cmake/app.cmake

```diff
@@ -1,6 +1,12 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+find_program(CCACHE "ccache")
+if(CCACHE)
+	set(CMAKE_C_COMPILER_LAUNCHER ${CCACHE})
```

Comment:
alignment fix here

Reply:
I've replace the tab with 4 spaces.

What about adding an .editorconfig to make sure these kinds of styles respected?

Reply:
> What about adding an .editorconfig to make sure these kinds of styles respected?

could be. We use astyle currently (not yet checking CMake files, might not even be able to)

Reply:
I was also wondering how we could check CMake files for our project, https://github.com/cheshirekow/cmake_format is a good candidate.

With .editorconfig, you "don't check anything", you just define the style. You then install the plugin in your favorite editor and it will use the style you've setup in your config file.

https://editorconfig.org/

It can be extension based or even file based if needed.

Quick example: https://github.com/ladislas/mbed-cmake-template/blob/master/.editorconfig

I can make a PR to master to start a conversation and add discuss the style definition.

Reply:
lets create an issue, and we can review.